### ISSUE TITLE
`continuation: true` with `locations: true` count library's `init` as a source  of promise

### ISSUE THREAD

**Issue by JaoodxD** [bengl/count-promises#1]
If `continuation` and `locations` both set to `true`, final result of `stop()` will show library's internal function `init` as a source of all promises.  

Environment:
* OS: Windows 10
* node: v20.6.1

Reproducable code example:  
```js
import startCounter from 'count-promises'
import { fetch, request } from 'undici'

const stop = startCounter({ locations: true, continuation: true })
const result = await fetch('http://www.example.com')
const result2 = await fetch('http://www.example.com')

const total = stop()
console.log(total) // { 'init (node_modules\\count-promises\\count-promises.js:53:7)': 128 }
```

**Comment by bengl** [comment]
Verified: This also happens with built-in fetch. Fix incoming.

**Comment by bengl** [comment]
@JaoodxD thanks for reporting! This is now fixed in v1.0.1.
